weather_mv/loader_pipeline/sinks.py

```diff
@@ -387,7 +387,7 @@ def open_dataset(uri: str,
     """Open the dataset at 'uri' and return a xarray.Dataset."""
     try:
         if is_zarr:
-            ds = xr.open_dataset(uri, engine='zarr', chunks=None)
+            ds: xr.Dataset = xr.open_dataset(uri, engine='zarr', **open_dataset_kwargs)
```

Comment:
Are `chunks=None` set by default? It's important to make sure this is the default behavior, since it will open the Zarr lazily.